hash/ibm5170_cdrom.xml

```diff
@@ -6169,6 +6169,49 @@ Includes DOS and Windows installation setup.
 		</part>
 	</software>
 
+	<!-- From the GOG release -->
+	<!-- Matches http://redump.org/disc/2431/ -->
+	<software name="strtrk25">
+		<description>Star Trek: 25th Anniversary (Enhanced CD-ROM)</description>
+		<year>1994</year>
+		<publisher>Interplay</publisher>
+		<info name="copy_protection" value="Manual required for map" />
+		<info name="developer" value="Interplay" />
+		<info name="language" value="English, French, German" />
```

Comment:
Please make the language info syntax like the others: 
<info name="language" value="English/French/German" />
For example, see the "Beneath a Steel Sky (Europe)" set.

Reply:
> Please make the language info syntax like the others: `<info name="language" value="English/French/German" />` For example, see the "Beneath a Steel Sky (Europe)" set.

Thanks :)